benchmarks/GeneratedHydratorBenchmark/AllPrivateClass.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace GeneratedHydratorBenchmark;
+
+class AllPrivateClass
+{
+    private $foo;
```

Comment:
A benchmark of a hydration of an object with default property values (mixed visibility) is also to be added